### ISSUE TITLE
Image Browser / Gallery

### ISSUE THREAD

**Issue by cmdr2** [cmdr2/stable-diffusion-ui#56]
I like the idea of an image browser for the images generated using the tool, which displays their metadata, with maybe a button to re-generate a particular old image (by setting the UI with that image's config).

https://github.com/cmdr2/stable-diffusion-ui/pull/29#issuecomment-1234494484

**Comment by i2chris** [comment]
Yes, the ability to jump back to any previous point would be great.  You wouldn't need to worry about losing a particular set of images or settings.  Would you store all the input images too?

**Comment by Ryder17z** [comment]
maybe it would be a decent idea to fork https://github.com/yfszzx/stable-diffusion-webui-images-browser and modify it to work with this one instead?
